modules/custom/social_ajax_comments/social_ajax_comments.module

```diff
@@ -193,7 +193,11 @@ function social_ajax_comments_ajax_render_alter(array &$data) {
         preg_match('/^#([a-z\-]+)$/', $item['selector'], $matches)
       ) {
         $content = &$item['data'];
-        $content = substr($content, strrpos(substr($content, 0, strpos($content, $matches[1])), '<'));
+        if (is_int($strpos_matches = strpos($content, $matches[1]))) {
+          if (is_int($strpos_substrng = strrpos(substr($content, 0, $strpos_matches), '<'))) {
+            $content = substr($content, $strpos_substrng);
+          }
+        }
```

Comment:
Can we move the assignment out of the if-statement to improve readability? Perhaps we should make this some kind of lint or style rule too. I always feel comprehension and refactorability of assignment-in-if is really poor.

Then we can also simply do `$strpos_matches !== FALSE` and `strpos_substrn !== FALSE` rather than `is_int` (since we know the return type is `int|FALSE`).

Reply:
Done